### PR TITLE
Add deployment scripts

### LINKED ISSUE #24: Create a script to deploy contracts

### PR DESCRIPTION
**Pull request by PraneshASP**
Fixes #24

### CODE REVIEW

File: scripts/DeployTransferVest.s.sol [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+    /// @dev Change these values before running the script!
+    address public owner = address(0xCAFE);
+    address public token = address(0xBEEF); 
+    address public sender = address(0xFACE); 
+
+    function run() public {
+        uint256 deployerPrivateKey = vm.envUint("DEPLOYER_KEY");
+        vm.startBroadcast(deployerPrivateKey);
+        address deployedInstance = address(new TransferVest(owner, sender, token));
+        console.log("TransferVest contract deployed at: ",deployedInstance, " for the token: ", token);
```
Comment (MerlinEgalite) [inline scripts/DeployTransferVest.s.sol:18]: I think the formatter hasn't been run here:
```suggestion
        console.log("TransferVest contract deployed at: ", deployedInstance, " for the token: ", token);
```

Comment (MerlinEgalite) [reply scripts/DeployTransferVest.s.sol:18]: Maybe I'll need to add in a pre-commit script

**Review by MerlinEgalite — CHANGES_REQUESTED** [review]
Nice thx!! 🎉 Just a small issue with formatting to resolve

File: scripts/DeployTransferVest.s.sol (hunk 1 above)
Comment (PraneshASP) [reply scripts/DeployTransferVest.s.sol:18]: Sure, happy to setup husky. Tracked via issue: https://github.com/MerlinEgalite/solvest/issues/26

**Comment by MerlinEgalite** [comment]
If you do the changes @PraneshASP I guess you can merge the PR

**Comment by PraneshASP** [comment]
> If you do the changes @PraneshASP I guess you can merge the PR

Sure. Will look into husky setup this week :+1:

File: scripts/DeployMintVest.s.sol [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+
+    /// @dev Change these values before running the script!
+    address public owner = address(0xCAFE);
+    address public token = address(0xBEEF); 
+
+    function run() public {
+        uint256 deployerPrivateKey = vm.envUint("DEPLOYER_KEY");
+        vm.startBroadcast(deployerPrivateKey);
+        address deployedInstance = address(new MintVest(owner,token));
+        console.log("MintVest contract deployed at: ",deployedInstance, " for the token: ", token);
```
[the remark this answers is not in the document]
Comment (PraneshASP) [reply scripts/DeployMintVest.s.sol:17]: Fixed here? https://github.com/MerlinEgalite/solvest/pull/25/commits/1a3753b9ac7270994a1a9c1dc91301770467b5ff The formatter doesn't complain. Tried running `forge fmt` @MerlinEgalite

**Review by MerlinEgalite — APPROVED** [review]

**Pull request merged**